test/unit/evault/modules/Vault/balancesWithInterest.t.sol

```diff
@@ -330,6 +330,43 @@ contract VaultTest_BalancesWithInterest is EVaultTestBase {
         assertEq(assetTST.balanceOf(user1), 100e18);
     }
 
+    function test_withdraw_interestLeftOver() public {
+        startHoax(address(this));
+        eTST.setInterestRateModel(address(new IRMTestFixed()));
+
+        startHoax(user1);
+        assetTST.mint(user1, 20e18);
+        eTST.deposit(20e18, user1);
+
+        startHoax(user3);
+        assetTST.mint(user3, 20e18);
+        assetTST.approve(address(eTST), type(uint256).max);
+        evc.enableController(user3, address(eTST));
+        eTST.borrow(1e18, user3);
+
+        assertEq(eTST.balanceOf(user1), 20e18);
+        assertEq(eTST.balanceOf(user3), 0);
+        assertEq(eTST.balanceOf(feeReceiver), 0);
+        assertEq(eTST.balanceOf(protocolFeeReceiver), 0);
+        assertEq(eTST.cash(), 19e18);
+
+        skip(10 days);
+
+        eTST.repay(type(uint256).max, user3);
+
+        startHoax(user1);
+        uint256 max = eTST.maxWithdraw(user1);
+        eTST.withdraw(max, user1, user1);
+
+        assertEq(eTST.balanceOf(user1), 0);
+        assertEq(eTST.balanceOf(user3), 0);
+        assertEq(eTST.balanceOf(feeReceiver), 0);
+        assertEq(eTST.balanceOf(protocolFeeReceiver), 0);
+
+        assertNotEq(eTST.cash(), 0);
+        assertApproxEqAbs(eTST.cash(), eTST.accumulatedFeesAssets(), 1000);
```

Comment:
we should first convert the fees and only then verify there's still some cash left